user/downloading-installing-upgrading/supported-versions.md

```diff
@@ -16,15 +16,15 @@ or minor release (see [Version Scheme]). The current release and past major
 releases are always available on the [Downloads] page, while all ISOs, including
 past minor releases, are available from our [download mirrors].
 
-| Qubes OS      | Start Date | End Date   | Status                |
-| ------------- | ---------- | ---------- | --------------------- |
-| Release 1     | 2012-09-03 | 2015-03-26 | Unsupported           |
-| Release 2     | 2014-09-26 | 2016-04-01 | Unsupported           |
-| Release 3.0   | 2015-10-01 | 2016-09-09 | Unsupported           |
-| Release 3.1   | 2016-03-09 | 2017-03-29 | Unsupported           |
-| Release 3.2   | 2016-09-29 | 2019-03-28 | Unsupported           |
-| Release 4.0   | 2018-03-28 | TBA        | Supported             |
-| Release 4.1   | TBA        | TBA        | [In development][4.1] |
+| Qubes OS    | Start Date | End Date   | Status                |
+| ----------- | ---------- | ---------- | --------------------- |
+| Release 1.0 | 2012-09-03 | 2015-03-26 | Unsupported           |
+| Release 2.0 | 2014-09-26 | 2016-04-01 | Unsupported           |
```

Comment:
We introduced x.y versioning only at 3.0 (https://www.qubes-os.org/doc/version-scheme/)

Reply:
Ah, I guess I shouldn't try to retcon those in. Fixed.